WordPressVIPMinimum/Sniffs/JS/StringConcatSniff.php

```diff
@@ -52,22 +52,18 @@ public function register() {
 	public function process( File $phpcsFile, $stackPtr ) {
 		$tokens = $phpcsFile->getTokens();
 
-		$nextToken = $phpcsFile->findNext( Tokens::$emptyTokens, ( $stackPtr + 1 ), null, true, null, true );
+		$nextToken = $phpcsFile->findNext( Tokens::$emptyTokens, $stackPtr + 1, null, true, null, true );
 
-		if ( T_CONSTANT_ENCAPSED_STRING === $tokens[ $nextToken ]['code'] ) {
-			if ( false !== strpos( $tokens[ $nextToken ]['content'], '<' ) && 1 === preg_match( '/\<\/[a-zA-Z]+/', $tokens[ $nextToken ]['content'] ) ) {
-				$data = [ '+' . $tokens[ $nextToken ]['content'] ];
-				$this->addFoundError( $phpcsFile, $stackPtr, $data );
-			}
+		if ( ( T_CONSTANT_ENCAPSED_STRING === $tokens[ $nextToken ]['code'] ) && false !== strpos( $tokens[ $nextToken ]['content'], '<' ) && 1 === preg_match( '/\<\/[a-zA-Z]+/', $tokens[ $nextToken ]['content'] ) ) {
+			$data = [ '+' . $tokens[ $nextToken ]['content'] ];
+			$this->addFoundError( $phpcsFile, $stackPtr, $data );
 		}
 
-		$prevToken = $phpcsFile->findPrevious( Tokens::$emptyTokens, ( $stackPtr - 1 ), null, true, null, true );
+		$prevToken = $phpcsFile->findPrevious( Tokens::$emptyTokens, $stackPtr - 1, null, true, null, true );
 
-		if ( T_CONSTANT_ENCAPSED_STRING === $tokens[ $prevToken ]['code'] ) {
-			if ( false !== strpos( $tokens[ $prevToken ]['content'], '<' ) && 1 === preg_match( '/\<[a-zA-Z]+/', $tokens[ $prevToken ]['content'] ) ) {
-				$data = [ $tokens[ $nextToken ]['content'] . '+' ];
-				$this->addFoundError( $phpcsFile, $stackPtr, $data );
-			}
+		if ( ( T_CONSTANT_ENCAPSED_STRING === $tokens[ $prevToken ]['code'] ) && false !== strpos( $tokens[ $prevToken ]['content'], '<' ) && 1 === preg_match( '/\<[a-zA-Z]+/', $tokens[ $prevToken ]['content'] ) ) {
```

Comment:
Should we remove the extra set of brackets enclosing `T_CONSTANT_ENCAPSED_STRING === $tokens[ $prevToken ]['code']`?